frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayProjectChanges.java

```diff
@@ -300,17 +300,19 @@ private boolean ensureSomethingTodoAndNoNullReleaseIdUsage() {
     }
 
     private Project prepareLicenseInfoHeaderTextInProject(Project project) {
+        Project modifiedProject = project.deepCopy();
```

Comment:
Instead of modifiedProject we could use newProject as in other deep copy variable names.